pkg/ctl/utils/describe_stacks.go

```diff
@@ -1,26 +1,40 @@
 package utils
 
 import (
+	"fmt"
+	"os"
+
 	"github.com/aws/aws-sdk-go/service/cloudformation"
 	"github.com/kris-nova/logger"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/printers"
 
 	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
-	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
 )
 
 func describeStacksCmd(cmd *cmdutils.Cmd) {
 	cfg := api.NewClusterConfig()
 	cmd.ClusterConfig = cfg
 
 	var all, events, trail bool
+	var output string
 
 	cmd.SetDescription("describe-stacks", "Describe CloudFormation stack for a given cluster", "")
 
 	cmd.CobraCommand.RunE = func(_ *cobra.Command, args []string) error {
 		cmd.NameArg = cmdutils.GetNameArg(args)
-		return doDescribeStacksCmd(cmd, all, events, trail)
+		switch output {
+		case "":
+			return doDescribeStacksCmd(cmd, all, events, trail, "")
+		case "yaml":
+			return doDescribeStacksCmd(cmd, all, events, trail, "yaml")
+		case "json":
+			return doDescribeStacksCmd(cmd, all, events, trail, "json")
+		default:
+			return fmt.Errorf("unknown output format: %s", output)
+		}
```

Comment:
You can construct a new `OutputPrinter` here and pass it to `doDescribeStacksCmd` where a `nil` value indicates default behaviour.  This validation is already performed in `printers.NewPrinter(output)`, but it accepts `table` so you'll have to validate for it. 

```suggestion

      var printer printers.OutputPrinter
		switch output {
		case printers.TableType:
			return errors.Errorf("output type %q is not supported", output)

		case "":

		default:
			var err error
			printer, err = printers.NewPrinter(output)
			if err != nil {
				return err
			}
		}
		return doDescribeStacksCmd(cmd, all, events, trail, printer)
```